Dockerfile

```diff
@@ -110,9 +111,10 @@ RUN chmod +x /home/steam/server/*.sh && \
 
 WORKDIR /home/steam/server
 RUN touch rcon.yaml crontab && \
-    chmod o+w rcon.yaml crontab && \
-    chown steam:steam -R /home/steam && \
-    chmod -R o+w /home/steam/steamcmd
+    mkdir -p /home/steam/Steam/package && \
+    rm -rf /tmp/dumps && \
+    chmod o+w rcon.yaml crontab /home/steam/Steam/package && \
+    chown steam:steam -R /home/steam
```

Comment:
Is this line still supposed to be there?

Reply:
(line 117)

Reply:
I recommend it because the files and directories we make in the build process are not owned by steam. Although it is not needed. Just makes it so the following are always owned by steam even when in rootless

```
/home/steam/server
/home/steam/server/start.sh
/home/steam/server/helper_install.sh
/home/steam/server/helper_functions.sh
/home/steam/server/compile-settings.sh
/home/steam/server/files
/home/steam/server/files/PalWorldSettings.ini.template
/home/steam/server/auto_reboot.sh
/home/steam/server/init.sh
/home/steam/server/discord.sh
/home/steam/server/crontab
/home/steam/server/rcon.yaml
```

Reply:
would `chown steam:steam -R /home/steam/server` work better?

Reply:
> would `chown steam:steam -R /home/steam/server` work better?

Yes. 

I'm getting 465.6MB with either one.